util/io_posix.cc

```diff
@@ -77,7 +140,11 @@ PosixSequentialFile::PosixSequentialFile(const std::string& fname, FILE* file,
     : filename_(fname),
       file_(file),
       fd_(fd),
-      use_direct_io_(options.use_direct_reads) {
+      use_direct_io_(options.use_direct_reads),
+      logical_sector_size_(kDefaultPageSize) {
+#ifdef OS_LINUX
```

Comment:
consider returning kDefaultPageSize from GetLogicalBufferSize() if OS_LINUX not defined, then you don't need to set it twice everywhere